libcaja-private/caja-file.c

```diff
@@ -2534,7 +2534,8 @@ update_info_internal (CajaFile *file,
 				(file->details->directory, file);
 
 			g_clear_pointer (&file->details->name, g_ref_string_release);
-			if (eel_strcmp (file->details->display_name, name) == 0) {
+			if (file->details->display_name &&
+			    eel_strcmp (file->details->display_name, name) == 0) {
```

Comment:
`name` isn't NULL, since the previous line `strcmp (file->details->name, name) != 0) {` doesn't trigger a SEGV, so you can use `strcmp` instead of `eel_strcmp`.

Please note that I cannot approve this change until I have verified that it solves an actual issue on a supported os, otherwise you may introduce a software regression or an incompatibility with current libs..

Reply:
Excellent point, I changed to just using strcmp.